nanvm-lib/src/big_numbers/big_uint.rs

```diff
@@ -21,28 +21,23 @@ impl BigUint {
     }
 
     pub fn normalize(&mut self) {
-        loop {
-            match self.value.last() {
-                Some(&last) if last == 0 => {
-                    self.value.pop();
-                }
-                _ => break,
-            }
+        while let Some(&0) = self.value.last() {
+            self.value.pop();
         }
     }
 
     pub fn is_one(&self) -> bool {
         self.len() == 1 && self.value[0] == 1
     }
 
-    pub fn is_zero(&self) -> bool {
-        self.len() == 0
-    }
-
     pub fn len(&self) -> usize {
         self.value.len()
     }
 
+    pub fn is_empty(&self) -> bool {
```

Comment:
I would keep `is_zero` name.

Reply:
I'll keep both since Clippy wanted is_empty because there is len.

Reply:
fixed